### PR TITLE
Add Minnows

### PR DESCRIPTION
**Pull request by TastyPumPum**
Add the fishing of minnows and the buying of raw sharks, anglers has to be equipped or in the bank to fish minnows as you need the outfit in game. Made it so crystal harpoon doesn't boost it. Rates were obtained from the osrs wiki and tested at each level the wiki had listed. @LuxrayElite helped with testing and coding.

-   [x] I have tested all my changes thoroughly.

### CODE REVIEW

**Comment by LuxrayElite** [comment]
Fun Collab of balancing 🤣

File: src/commands/Minion/fish.ts [hunk 1]
```diff
@@ -67,6 +67,16 @@ export default class extends BotCommand {
 			return msg.channel.send('You need at least 15 Agility and Strength to do Barbarian Fishing.');
 		}
 
+		if (
+			fish.name === 'Minnow' &&
+			(!msg.author.hasItemEquippedOrInBank('Angler top') ||
+				!msg.author.hasItemEquippedOrInBank('Angler waders') ||
+				!msg.author.hasItemEquippedOrInBank('Angler boots') ||
+				!msg.author.hasItemEquippedOrInBank('Angler hat'))
+		) {
```
Comment (gc) [inline src/commands/Minion/fish.ts:76]: use `[].some` instead of this.

**Review by gc — CHANGES_REQUESTED** [review]
looks alright, is it tested by testers?

**Comment by TastyPumPum** [comment]
> looks alright, is it tested by testers?

It has been tested throughly by myself, lux has helped test and code it. I can get it tested extensively in ironcord.

**Comment by TastyPumPum** [comment]
> looks alright, is it tested by testers?

The scaled time per fish has now been done on a curve basis using the information provide on the wiki, this has been capped at the scale for 99 due to BSO levelling. The cap can be removed if needed. 

This has been tested a lot by myself, with others testing and helping.

**Pull request merged**